osu.Game.Rulesets.Taiko/Difficulty/Preprocessing/Rhythm/IHasInterval.cs

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+namespace osu.Game.Rulesets.Taiko.Difficulty.Preprocessing.Rhythm
+{
+    /// <summary>
+    /// The interface for hitobjects that provide an interval value.
+    /// </summary>
+    public interface IHasInterval
+    {
+        double Interval { get; }
```

Comment:
What's an interval?

Reply:
Interval referring to the changes in rhythm (a change in interval // spacing of an object), and being outputted eventually as a ratio. Originally having a tiny class for solely an interface was questionable, however it was the best outcome I could think of at the time.